extensions/amp-story/1.0/localization.js

```diff
@@ -81,7 +81,7 @@ export let LocalizedStringBundleDef;
  * Language code used if there is no language code specified by the document.
  * @const {string}
  */
-const DEFAULT_LANGUAGE_CODE = 'default';
```

Comment:
I think this is actually my fault for leaving detail out of the bug (and the comment above is also wrong), but there are actually two cases handled by this code:

1. The publisher did not specify a language
2. The publisher specified a language that we don't support

I think in the ideal state, we want to fall back to "en" for (1), but fall back to "default" for (2), so we may need to figure out how to split this code path into two cases.

Reply:
Thanks for the help! That makes a lot of sense. I updated it to do this functionality I think.